### PR TITLE
Feature/fixs video issues

### PR DESCRIPTION
**Pull request by francis-guardi**
fixed video issues

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -33,65 +33,87 @@ export default function Home() {
...
-        }
-      })
-      .catch(function (error) {
-        console.log(error);
-      });
-  }, [navigate]);
+    if (response == null) {
+      console.log("No events found!");
+    } else {
+      const notification_list = updateCameraNameInNotificatios(
```
Comment (felmoreno1726) [inline src/pages/index.js:56]: Typo: notifications (missing s)

Comment (felmoreno1726) [reply src/pages/index.js:56]: I mean. Missing n

File: src/pages/index.js [hunk 2]
```diff
@@ -33,65 +33,87 @@ export default function Home() {
...
+      setEvents(notification_list);
+    }
+  })
+  .catch(function (error) {
+    console.log(error);
+  });
+
+}
+
+  function updateCameraNameInNotificatios(notificationList, listOfCameras) {
```
Comment (felmoreno1726) [inline src/pages/index.js:69]: notificatio(N)s

File: src/pages/index.js (hunk 1 above)
Comment (francis-guardi) [reply src/pages/index.js:56]: Do you want to replace notification_list  for notification**s**_list I use in singular beacuse of List in the end!

File: src/pages/index.js (hunk 2 above)
Comment (francis-guardi) [reply src/pages/index.js:69]: Do you want to replace notification_list for notifications_list I use in singular beacuse of List in the end!

**Review by felmoreno1726 — APPROVED** [review]
Potente

**Pull request merged**